docs/dashboards/types-of-components.md

```diff
@@ -46,3 +60,38 @@ Another type of component type, that allows you to visualize key performance ind
 You can define the threshold to change the style of the component, when one value exceeds it and some other useful features to better show what is important to you.
 
 [Here is the example](http://utils.highcharts.local/samples/#view/dashboards/demos/dashboards-component-kpi)
+
+Code snippet:
+``` JS
+   {
+        cell: 'kpi-00',
+        type: 'KPI',
+        title: 'Average revenue',
```

Comment:
How about other params like:
-  `subtitle`
-  `chartOptions`
-  `valueFormat`
-  `valueFormatter`

Reply:
This is more general documentation, all these properties are going to be in API